src/test/shared/credentials/userCredentialsUtils.test.ts

```diff
@@ -108,33 +109,48 @@ describe('UserCredentialsUtils', function () {
             }
             await UserCredentialsUtils.generateCredentialsFile(creds)
 
-            const sharedConfigFiles: SharedConfigFiles = await loadSharedConfigFiles()
+            const sharedConfigFiles = await loadSharedConfigFiles({ credentials: Uri.file(credentialsFilename) })
             assert(typeof sharedConfigFiles === 'object', 'sharedConfigFiles should be an object')
             const profiles = sharedConfigFiles.credentialsFile
             assert(typeof profiles === 'object', 'profiles should be an object')
-            assert(profiles[profileName], 'profiles should be truthy')
+            const profile = profiles[profileName]
+            assert.ok(profile)
             assert.strictEqual(
-                profiles[profileName].aws_access_key_id,
+                profile.aws_access_key_id,
                 creds.accessKey,
-                `creds.accessKey: "${profiles[profileName].aws_access_key_id}" !== "${creds.accessKey}"`
+                `creds.accessKey: "${profile.aws_access_key_id}" !== "${creds.accessKey}"`
             )
             assert.strictEqual(
-                profiles[profileName].aws_secret_access_key,
+                profile.aws_secret_access_key,
                 creds.secretKey,
-                `creds.secretKey: "${profiles[profileName].aws_access_key_id}" !== "${creds.secretKey}"`
+                `creds.secretKey: "${profile.aws_access_key_id}" !== "${creds.secretKey}"`
             )
             await fs.access(credentialsFilename, fs.constants.R_OK).catch(_err => assert(false, 'Should be readable'))
             await fs.access(credentialsFilename, fs.constants.W_OK).catch(_err => assert(false, 'Should be writeable'))
         })
     })
 
+    describe('loadSharedConfigFiles', function () {
+        it('normalizes fields by making them lowercase', async function () {
+            const credentialsFilename = path.join(tempFolder, 'credentials-generation-test')
+            const profileName = 'someRandomProfileName'
+
+            createCredentialsFile(credentialsFilename, [profileName])
+
+            const sharedConfigFiles = await loadSharedConfigFiles({ credentials: Uri.file(credentialsFilename) })
+            const profiles = sharedConfigFiles.credentialsFile
+            assert.strictEqual(profiles[profileName]?.region, 'us-weast-3')
+        })
+    })
+
     function createCredentialsFile(filename: string, profileNames: string[]): void {
         let fileContents = ''
 
         profileNames.forEach(profileName => {
             fileContents += `[${profileName}]
 aws_access_key_id = FAKEKEY
 aws_secret_access_key = FAKESECRET
```

Comment:
maybe use random casing here?
```suggestion
aws_SecRet_aCCess_key = FAKESECRET
```